### PR TITLE
Python dev script

### PR DESCRIPTION
**Pull request by ref-humbold**
Replaced shell script with Python script based on argparse and subprocess modules

### CODE REVIEW

**Comment by Bozydarek** [comment]
I think that `argcomplete` might be useful (https://argcomplete.readthedocs.io/en/latest/)

File: tools/dev.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+WEB_CONTAINER_NAME = f"{PROJECT_NAME}_web_1"
+DB_CONTAINER_NAME = f"{PROJECT_NAME}_db_1"
+FILE_SYSTEM_NOTE = f"({C_yellow}note:{C_normal} this may create files on host fs with root permissions)"
+
+
+def shell_run(command):
+    if args.debug:
+        print(f"{C_white}** {C_yellow}{command}{C_white} **{C_normal}")
+
+    subp.run(command, shell=True)
```
Comment (Bozydarek) [inline tools/dev.py]: `shell=True` should not be used. Please consider using other python functions to get same results, like
`glob`, `fnmatch`, `os.walk()`, `os.path.expandvars()`, `os.path.expanduser()`, and `shutil`

File: tools/dev.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    setup()
+elif args.command in ["shutdown", "sd"]:
+    docker_compose("down")
+elif args.command in ["test", "t"]:
+    docker_python("test")
+elif args.command in ["shell", "sh"]:
+    if args.shell == "bash":
+        docker_exec("/bin/bash", WEB_CONTAINER_NAME)
+    elif args.shell in ["postgres", "psql"]:
+        docker_exec("psql -U zosia", DB_CONTAINER_NAME)
```
[the remark this answers is not in the document]
Comment (ref-humbold) [reply tools/dev.py]: In these cases I can make the subparsers required, so as to throw an error.

File: tools/dev.py (hunk 1 above)
Comment (ref-humbold) [reply tools/dev.py]: I suppose we can get rid of `shell=True`, because `docker` and `docker-compose` are recognized as an executable.

**Pull request merged**